.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true,
+  },
+  rules:{
+    "prettier/prettier" : "error",
+    "eqeqeq": "error"
+  },
+  extends: ["prettier"],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "tsconfig.json",
+    sourceType: "module",
+  },
+  plugins: ["prettier"],
```

Comment:
Should `parser: "@typescript-eslint/parser"` and `parserOptions:` be in `overrides`? For me, it looks that it's the configuration of Typescript instead of the Javascript.

Reply:
I am not sure about this either but IMO we a have typescript project where we are allowing JS files in the project so
TS is on top maybe @kelson42 can give his opinion too

Reply:
Ok, let’s keep it.